torch/csrc/distributed/rpc/process_group_agent.h

```diff
@@ -7,12 +7,13 @@
 #include <atomic>
 #include <thread>
 
+#include <fmt/format.h>
```

Comment:
even if we use libfmt, does this need to be included in a public header?

Reply:
Moved it to the cpp